internal/jujuapi/access_control.go

```diff
@@ -58,6 +58,23 @@ func (r *controllerRoot) AddGroup(ctx context.Context, req apiparams.AddGroupReq
 	return resp, nil
 }
 
+// GetGroup retrieves a group within JIMMs DB for reference by OpenFGA.
```

Comment:
This docstring could be simpler 
```suggestion
// GetGroup returns group information based on a group ID.
```

Reply:
Other methods have godocs like this so I just went with the convention but didn't like it too. We can just simplify all of them then.